config/locales/en/shared/schools/mentors.yml

```diff
@@ -0,0 +1,10 @@
+en:
+  shared:
+    schools:
+      mentors:
+        can_not_remove:
+          title: You can not remove this mentor
+          description: |
+            %{mentor_name} cannot be removed from your organisation because they are assigned to placements.
+
+            To remove %{mentor_name} as a mentor you must remove them from any current placements.
```

Comment:
This looks like Placements-specific content.

Reply:
As with the views, the shared locales namespace is not strictly service agnostic.

For example: [config/locales/en/shared/organisations/grant_funding.yml](https://github.com/DFE-Digital/itt-mentor-services/blob/15e7898301603085bdbdd39567358c5dcea72728/config/locales/en/shared/organisations/grant_funding.yml)